include/sdf/Element.hh

```diff
@@ -101,6 +101,7 @@ namespace sdf
 
     /// \brief Set the requirement type.
     /// \param[in] _req Requirement type for this element:
+    /// -1: Deprecated.
```

Comment:
BTW This smells funky. I understand motivation is to have easy translation btw schema / XSD and here, but may be better as an enum at some point. Lemme know if an issue is useful.

Reply:
Yeah, an enum would be good. We have a lot of places in the code where we're doing string comparison. But, I think that's a bigger change than what I'd like for this PR. An issue would be great!

Reply:
Done! https://github.com/osrf/sdformat/issues/553